### ISSUE TITLE
FakeSMC/GPUSensors modification source code unavailable?

### ISSUE THREAD

**Issue by metacollin** [KGP/X299-EFI-Folder-Distributions#1]
Hi, I can't seem to find the source code anywhere that has your changes to FakeSMC/GPUSensors that enabled Vega/Radeon VII temperature sensor support.  I am probably just not looking hard enough, but could you help me out and provide a link to the source code?  You might even try doing a pull request to whichever FakeSMC repo is being maintained these days (RehabMan's on Bitbucket I think)?

**Comment by skyflying5** [comment]
https://github.com/RehabMan/OS-X-FakeSMC-kozlek
here？

**Comment by metacollin** [comment]
Yes, that's the repo I was suggesting he do a pull request too.  It has semi-broken Vega10 support (reports temperature only, not power and causes excessive boot times due to sensor probing timeouts on certain Vega 56 and 64 cards).  It does not have Vega20 (Radeon VII) support at all.  

In KGP's X299 EFI repo (the one this issue has been raised on), KGP has a compiled version of the radeon sensors kext that he has added Vega20 support (and possibly fixed/better Vega 10 support? Who knows) but he has not, as near as I can tell, provided the source code or otherwise made his changes publicly available.
